WordVectors/python/analysis_notebook.ipynb

```diff
@@ -0,0 +1,892 @@
+    "\n",
+    "Now that we have our x and y axes, we can plot them. We start by declaring a variable `fig`, short for figure, which will represent our plot. We set the size of our particular figure to `(8, 6)`, but you should feel free to play around with the sizing on your own. Different sizes may work better with your data depending on its size. Then, we declare a variable, `ax`, which is short of axis. The `ax` variable will allow us to label each axis as well as access individual points on the plot, making labeling much easier.\n",
+    "\n",
+    "We use the `ax` variable to tell the computer that we want a 2D graph, and that we want to produce a scatter plot using the values associated with the x and y axes, each of which contains one of our principal components. \n",
+    "\n",
+    "The `for` loop which follows is how we are going to label our points. Using the built in function `.text()` we tell the computer to visit each point on the plot, and assign the corresponding label from our `labels` list to that point. I've set the font to be italic, blue, and the font size to 10, but these values can be adjusted to your liking. You can also comment out the `for` loop if you want to look at the PCA graph without the labels. \n",
+    "\n",
+    "Finally, we label the x and y axes. For this walkthrough, I have simply labeled them \"x axis\" and \"y axis\" though these can be changed to whatever would be useful to you. \n",
+    "\n",
+    "We end the function by telling the computer to show us the resulting plot by using the built in `.show()` function. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a scatter plot of the projection\n",
+    "\n",
+    "# set the points for the x_axis by grabbing all the items in column one of the results variable\n",
+    "x_axis = result[:,0]\n",
+    "\n",
+    "# set the points for the y_axis by grabbing all the items in column two of the results variable\n",
+    "y_axis = result[:,1]\n",
+    "\n",
+    "# create a new figure of size (8, 6)\n",
+    "fig = plt.figure(figsize=(8, 6)) # you can change this size\n",
+    "\n",
+    "# create a variable for the axes and let the computer know we want a 2D plot\n",
+    "ax = fig.add_subplot(111)\n",
+    "\n",
+    "# scatterplot the points by using the x_axis variable as X and the y_axis variable as Y\n",
+    "ax.scatter(x_axis, y_axis)\n",
+    "\n",
+    "# iterate through our labels list\n",
+    "    for i in range(len(labels)):\n",
+    "        \n",
+    "        # assign labels to the points and make those labels italic, blue, and size 10\n",
+    "        ax.text(x_axis[i], y_axis[i], labels[i], style ='italic',\n",
+    "        fontsize = 10, color =\"blue\")  # you can change these values\n",
+    "        \n",
```

Comment:
The indentation’s off here, though after fixing it I realized that this code isn’t meant to stand alone, it’s meant to illustrate the descriptions of what the code’s doing. I’d like to see a little more signposting at the beginning of this section so that it’s more apparent that these code blocks aren’t intended to be run, and will actually break if you try.

It would be more work, but I’d love to see each code block as something that someone could run. You could start with one model, `model`, and walk through the code to make a scatterplot of just that model. Then, when people see how it all fits together, you can extrapolate out into a function that takes all that code and makes it repeatable on any model.

I find people are more comfortable taking the simpler route at first; being able to build on that hard-won knowledge is empowering and invigorating.